lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -295,6 +295,14 @@ def supports_network_tokenization?
         true
       end
 
+      def check_external_account_limit(account)
+        auth_header = { 'Authorization': "Bearer #{options[:login]}" }
+        url = "#{live_url}accounts/#{CGI.escape(account)}/external_accounts"
+        accounts_response = JSON.parse(ssl_get("#{url}?limit=100", auth_header))
+        to_delete = accounts_response['data'].reject { |ac| ac['default_for_currency'] }
+        ssl_request(:delete, "#{url}/#{to_delete.first['id']}", nil, auth_header)
+      end
```

Comment:
```suggestion
      def delete_latest_test_external_account(account)
        return unless test?
        auth_header = { 'Authorization': "Bearer #{options[:login]}" }
        url = "#{live_url}accounts/#{CGI.escape(account)}/external_accounts"
        accounts_response = JSON.parse(ssl_get("#{url}?limit=100", auth_header))
        to_delete = accounts_response['data'].reject { |ac| ac['default_for_currency'] }
        ssl_request(:delete, "#{url}/#{to_delete.first['id']}", nil, auth_header)
      end
```

A few suggestions for explicitness and safety.

Reply:
Suggestions added and working fine, let me know if is there anything else to improve.